### ISSUE TITLE
SSH Activity (4007) Class Enhancements

### ISSUE THREAD

**Issue by Aniak5** [ocsf/ocsf-schema#794]
Related to #728. Expand ssh activity class to contain SSHD server information  such as
- Authentication Type (password, public key, etc)
- TTY session command logging

**Comment by Aniak5** [comment]
TTY session command logging will be addressed by @mr-splunk in another issue.

**Comment by mr-splunk** [comment]
Thanks @Aniak5. Great Network Activity discussion today. To recap, looking deeper at #728 we uncovered. 3 primary topics:

1. We looked within *nix `sshd_config` to gather the Authentication Types: https://man.openbsd.org/sshd_config. @Aniak5 is going to look at working these in to the [SSH Activity](https://schema.ocsf.io/1.0.0/classes/ssh_activity?extensions=) class.
2. `tty` is specific to *nix hosts, as well as it's younger sibling `pty`. These two attributes have proven useful for detection engineers, so let's account for them. I've created pr <placeholder> to address this.
3. As for SSHD config changes - this sparked much conversation around whether such actions fall under `File System Activity` or even `Device Config State`. Also, we noticed that the `activity_ids` for every class in the `Discovery` category is the same, and folks are questioning whether that is the optimal approach.
